modules/ROOT/pages/to-download-an-asset.adoc

```diff
@@ -8,6 +8,8 @@ Download lets you download a copy of the asset, or in the case of a Custom asset
 
 After creating a REST asset, Exchange generates a Mule 3 connector and a Mule 4 connector. Exchange also generates an OAS version for RAML assets, and a RAML version for OAS assets. After you create a new REST asset it may take a few minutes for these assets to appear on the Download menu.
 
+When you change the asset name, description, or tags of the REST asset, Exchange updates these attributes for the generated connectors.
```

Comment:
Same comment as https://github.com/mulesoft/docs-release-notes/pull/1884. Why tags? Why not labels? If it's referring to user defined tags, then what about categories? 

@mcampo

Reply:
same answer, let's replace "tags" with "labels and categories"

Reply:
Done. Thank you @mcampo !